docs: vale DITA compliance for installing and multitenancy assemblies (#8323)
Apicurio/apicurio-registry: docs/modules/ROOT/pages/getting-started/assembly-installing-registry-storage-openshift.adoc

You can install {registry} in an existing Kafka cluster or create a new Kafka cluster, depending on your environment.

.Prerequisites

> * Ensure that you have an OpenShift cluster with cluster administrator access.

Reviewer: ```suggestion
* An OpenShift cluster with cluster administrator access is configured.
```

Author: Applied, thanks. Updated all prerequisites in this file to match.

> * Ensure that you have installed {registry}. See {installing-the-registry-openshift}.

Reviewer: ```suggestion
* The {registry} is installed. See {installing-the-registry-openshift}.
```

Author: Applied, thanks.

> * Ensure that you have installed {kafka-streams}. See xref:installing-kafka-streams-operatorhub_{context}[].

Reviewer: ```suggestion
* {kafka-streams} is installed. See xref:installing-kafka-streams-operatorhub_{context}[].
```

Author: Applied, thanks.

|===

[role="_additional-resources"]
.Additional resources
* xref:getting-started/assembly-configuring-the-registry.adoc[]

// Metadata created by nebel
//
// ParentAssemblies: assemblies/getting-started/as_installing-the-registry.adoc

// proc-installing-postgresql-operatorhub
[id="installing-postgresql-operatorhub_{context}"]
:_mod-docs-content-type: PROCEDURE
== Installing a PostgreSQL database from the OpenShift OperatorHub

[role="_abstract"]
If you do not already have a PostgreSQL database Operator installed, you can install a PostgreSQL Operator on your OpenShift cluster from the OperatorHub. The OperatorHub is available from the OpenShift Container Platform web console. It provides an interface for cluster administrators to discover and install Operators. For more details, see link:{LinkOpenShiftIntroOperator}[{NameOpenShiftIntroOperator}].

.Prerequisites

> * Ensure that you have cluster administrator access to an OpenShift cluster.

Reviewer: ```suggestion
* Cluster administrator access to an OpenShift cluster is configured.
```

Author: Applied, thanks.

// Metadata created by nebel
// ParentAssemblies: assemblies/getting-started/as_installing-the-registry.adoc

// proc-setting-up-postgresql-storage
[id="setting-up-postgresql-storage_{context}"]
:_mod-docs-content-type: PROCEDURE
== Configuring {registry} with PostgreSQL database storage on OpenShift

[role="_abstract"]
Configure storage for {registry} on OpenShift using a PostgreSQL database Operator. You can install {registry} in an existing database or create a new database, depending on your environment. The following example uses the PostgreSQL Operator by Dev4Ddevs.com.

.Prerequisites

> * Ensure that you have an OpenShift cluster with cluster administrator access.

Reviewer: ```suggestion
* An OpenShift cluster with cluster administrator access is configured.
```

Author: Applied, thanks.

> * Ensure that you have installed {registry}. See {installing-the-registry-openshift}.

Reviewer: ```suggestion
* The {registry} is installed. See {installing-the-registry-openshift}.
```

Author: Applied, thanks.

> * Ensure that you have installed a PostgreSQL Operator on OpenShift. For example, see xref:installing-postgresql-operatorhub_{context}[].

Reviewer: ```suggestion
* PostgreSQL Operator is installed on OpenShift. For example, see xref:installing-postgresql-operatorhub_{context}[].
```

Author: Applied — used "A PostgreSQL Operator is installed on OpenShift."
